djstripe/settings.py

```diff
@@ -54,6 +54,17 @@ def get_callback_function(setting_name, default=None):
 subscriber_request_callback = get_callback_function("DJSTRIPE_SUBSCRIBER_MODEL_REQUEST_CALLBACK",
                                                     default=(lambda request: request.user))
 
+
+def _get_idempotency_key(object_type, action, livemode):
+    from .models import IdempotencyKey
+    action = "%s:%s" % (object_type, action)
+    idempotency_key, _ = IdempotencyKey.objects.get_or_create(action=action)
```

Comment:
``_created`` -- shows that it's not used, but still tells the next dev what's returned (without looking up the docs)